sender-sqs/src/test/java/zipkin/sender/awssqs/AwsBufferedSqsSenderTest.java

```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package zipkin.reporter.awssqs;
+package zipkin.sender.awssqs;
```

Comment:
actually the package is correct (or at least congruent with others). sender is zipkin.reporter.Sender

the maven artifact just says that this only includes a sender and saying zipkin-reporter-sender-sqs is just crap :)